vimiv/api/settings.py

```diff
@@ -333,6 +409,12 @@ def __str__(self) -> str:
 read_only = BoolSetting(
     "read_only", False, desc="Disable any commands that are able to edit files on disk"
 )
+image_order = ImageOrderSetting(
```

Comment:
We would then always create a `OrderSetting` and pass the additional options for images / directories here.

Reply:
Implemented in 78ad67e7879ea4250b85e8b3e62b4c401b39404a.